CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing guide
+
+We're always looking for new contributors and are happy to help you get started! Some ideas:
+
+- Add a test for a function.
+- Improve a function's documentation.
+- Work on any issue tagged [good first issue](https://github.com/remeda/remeda/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+- Work on any issue tagged [help wanted](https://github.com/remeda/remeda/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
+
+## Getting started
+
+Requirements:
+
+- Node.js `>=18.20.0`
+- npm `>=10.7.0`
```

Comment:
This isn't accurate, we use nvm and should guide people to rely on nvm to use the version in nvmrc so that we can "lock" versions if we need to (we currently use the latest 22 version I think)